helm/templates/clusterrole.yaml

```diff
@@ -1,4 +1,5 @@
 # Add permission to list the POD for in cluster PODs
+{{- if .Values.cno }}

```

Comment:
This is not correct, this role is always needed as it gives us the ability to read the PODs as well as other things like calico and cilium CRD that are technically not needed all at the same time but well that is how it was done.